models/datasets/generic/ds__invoicing_transactions_insurance.yml

```
@@ -0,0 +1,40 @@
+version: 2
+
+models:
+  - name: ds__invoicing_transactions_insurance
+    description: Invoicing Transactions Insurance
```

Sentence casing for consistency?
```suggestion
    description: Invoicing transactions - insurance
```